Adding API AM production warnings to relevant pages (#2849)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/concepts/api-access-management/index.md

# API Access Management with Okta

> > **Note**: [Okta’s Developer Edition](/signup/) makes most key developer features available by default for testing purposes. Please note that API Access Management is an optional add-on in production.

Reviewer: We don't use "Please" and saying note is sort of redundant considering the text is IN a note, right?

Do you need to add: — a requirement for Custom Authorization Servers — to this one?

Also - at the end of the note, should we provide them a place to go if they want to add the API Access Management feature to their production environment?

```suggestion
> **Note**: [Okta’s Developer Edition](/signup/) makes most key developer features available by default for testing purposes. [API Access Management](/docs/concepts/api-access-management/) is an optional add-on for production environments.
```

Author: I've updated the wording according to feedback from you and Kelsey in PMM. The things I haven't done are as follows:

* I didn't remove "Note:" because a lot of our other note boxes have "Note:", and I like the way they look with "Note:" at the start. I think I am just used to note boxes being like that — same as we did in my last two companies. Do we have a house style guideline to govern this? If not, I'd be happy to agree one and go with it.
* I didn't add " — a requirement for Custom Authorization Servers —" to this page, as this page is specifically about API AM, and its uses are explained below in the article. The same is not true for the other pages, so a bit of context is helpful there.
* I've deliberately not included a link to where to find out more information about adding API AM, as I thought that would get too close to providing information the feature's price, or SKU, and I've been told that we have an explicit policy not to mention such things in the docs. I'm assuming that any reader that is close to wanting to use API in production will just ask their Okta sales rep?

Reviewer: @chrisdmills-okta - what I meant about removing the word "note" was the "Please note that" that I removed in each of  the notes -- not actually removing Note: Sorry if that wasn't clear. haha

Author: @susanharper-okta ah, OK! Sorry, I totally misunderstood that ;-) In that case we are all good. the new wording removes this.
